steps/plandek-deployment/2.0.0/step.yml

```diff
@@ -0,0 +1,162 @@
+title: Plandek - deployment
+summary: |
+  Enables you to track and visualize build and deployment events within Plandek's metrics and dashboards.
+description: |
+  Wrapper implementing the Plandek /deployment POST API. Used to complete a deployment within Plandek for a given release cycle.
+  More details can be found here: [https://api.plandek.com/deployments/v1/ui/](https://api.plandek.com/deployments/v1/ui/)
+website: https://github.com/secretescapes/bitrise-step-plandek-deployment
+source_code_url: https://github.com/secretescapes/bitrise-step-plandek-deployment
+support_url: https://github.com/secretescapes/bitrise-step-plandek-deployment/issues
+published_at: 2023-08-30T10:32:24.454204+02:00
+source:
+  git: https://github.com/secretescapes/bitrise-step-plandek-deployment.git
+  commit: 7228ccb6d6a71141aa2ab534f2b7f6b86895e970
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
+  - name: curl
+  apt_get:
+  - name: jq
+  - name: curl
+is_always_run: true
+inputs:
+- api_token: null
+  opts:
+    description: |
+      Plandek API token.
+    is_expand: true
```

Comment:
NOTE: `is_expand` is `true` by default, so there is no need to explicitly specify it, also just wanted to leave it as a reminder that it is on `true` even if it is not specified in the step.yml.